frontend/src/components/common/RenderCheckbox.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Form, Checkbox } from 'antd';
+
+/**
+ * @constant RenderCheckbox - Ant Design `Checkbox` component for redux-form.
+ */
+
+const propTypes = {
+  id: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+  label: PropTypes.string,
+  meta: PropTypes.object,
+};
+
+const RenderCheckbox = ({
+  id,
+  input,
+  label,
+  meta: { touched, error},
+}) => (
+  <Form.Item 
+  validateStatus={(touched ? (error && 'error') : '')}
+>
```

Comment:
Looks like this should be vertically aligned

Reply:
Good catch, alignment was very off.